Composer/packages/client/src/components/QnA/CreateQnAFromQnAMaker.tsx

```diff
@@ -22,7 +25,8 @@ const formConfig: FieldConfig<CreateQnAFromQnAMakerFormData> = {
 };
 
 export const CreateQnAFromQnAMaker: React.FC<CreateQnAFromFormProps> = (props) => {
-  const { onChange, qnaFiles, initialName } = props;
+  const { onChange, qnaFiles, initialName, onNext } = props;
+  const [signedInAccount, setSignedInAccount] = useState('');
```

Comment:
this is not an account, it's a string showing the username/email right? my suggestion: accountEmail